### PR TITLE
 Fix type of style colors

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

**Review by mrousavy — APPROVED** [review]

**Comment by jakub-gonet** [comment]
[CI fails.](https://travis-ci.org/github/software-mansion/react-native-reanimated/jobs/749029715#L222-L233)

**Comment by wcandillon** [comment]
@jakub-gonet thanks, should be green now.

**Review by Szymon20000 — APPROVED** [review]

File: react-native-reanimated.d.ts [hunk 1]
```diff
@@ -26,7 +26,8 @@ declare module 'react-native-reanimated' {
     ScrollView as ReactNativeScrollView,
     NativeScrollEvent,
     NativeSyntheticEvent,
-    ColorValue
+    ColorValue,
+    OpaqueColorValue,
```
Comment (jakub-gonet) [inline react-native-reanimated.d.ts]: Unused.

File: tsconfig.json [hunk 2]
```diff
@@ -7,8 +7,10 @@
     "skipLibCheck": true,
     "allowSyntheticDefaultImports": true,
     "moduleResolution": "node",
-    "lib": ["es2015"],
-    "skipLibCheck": true
+    "lib": ["es6", "es2015"],
```
Comment (jakub-gonet) [inline tsconfig.json:10]: Those are actually the same, let's leave es6 here.

File: tsconfig.json [hunk 3]
```diff
@@ -7,8 +7,10 @@
...
+    "skipLibCheck": true,
```
Comment (jakub-gonet) [inline tsconfig.json:11]: Duplicated prop (see above allowSyntheticDefaultImports prop).

Comment (karol-bisztyga) [reply tsconfig.json]: fixed 5538f62

File: tsconfig.json (hunk 2 above)
Comment (karol-bisztyga) [reply tsconfig.json]: fixed 5538f62

**Review by jakub-gonet — APPROVED** [review]

**Pull request merged**